### DISCUSSION TITLE
How to prevent blocking from YouTube?

### DISCUSSION THREAD

**Discussion by grifgrif** [dhowe/AdNauseam#2425] [Q&A] [answered]
Hello,
When I go on YouTube, it blocks after 3 videos.
Do you have a way to block it or prevent it?

Screenshot of what happens:
![screen](http://i.discord.fr/PJs3.png)

**Comment by phlave** [comment, marked as answer]
access the ublock origin panel and update the filters, especially the "quick fixes" list

**Reply by grifgrif** [reply]
Thank you for the answer!
I've manually updated them and it seems to work.

There is an automatic one? How can we make sure it updates often?

**Reply by phlave** [reply]
I'm not sure, honestly. There's an active discussion in a pinned thread in the uBlockOrigin subreddit, but I'm not super up-to-date on the discussion.

**Comment by mneunomne** [comment]
For youtube issues please go to this issues page: https://github.com/dhowe/AdNauseam/issues/2544
